src/packages/utils/ratlib.py

```diff
@@ -11,15 +11,15 @@
 This module is built on top of the Pydle system.
 
 """
-import re
-import humanfriendly
 import datetime
-from datetime import timedelta
+import re
+from dataclasses import dataclass
 from enum import Enum
 from math import isclose, sqrt
-from uuid import UUID
 from typing import Optional
-from dataclasses import dataclass
+from uuid import UUID
+
+import humanfriendly
```

Comment:
We should fix this while we're at it as well.
```
src/packages/utils/ratlib.py:17: [C0411(wrong-import-order), ] standard import "from enum import Enum" should be placed before "from dataclasses import dataclass"
src/packages/utils/ratlib.py:18: [C0411(wrong-import-order), ] standard import "from math import isclose, sqrt" should be placed before "from dataclasses import dataclass"
src/packages/utils/ratlib.py:19: [C0411(wrong-import-order), ] standard import "from typing import Optional" should be placed before "from dataclasses import dataclass"
src/packages/utils/ratlib.py:20: [C0411(wrong-import-order), ] standard import "from uuid import UUID" should be placed before "from dataclasses import dataclass"
src/packages/utils/ratlib.py:22: [C0411(wrong-import-order), ] third party import "import humanfriendly" should be placed before "from dataclasses import dataclass"
```

Reply:
I have a fix for that, but I felt it was out of scope. the import order here is actually correct, but pylint doesn't recognize `dataclasses` as a standard import. the fix is a 1 liner in the `pylint.rc` which i have queued for another PR.

Reply:
Fair enough, I see it now. Rescinded!